plugins/feeds/public/wiz_cloud_threat_landscape.py

```diff
@@ -0,0 +1,275 @@
+import logging
+import re
+from datetime import datetime, timedelta
+from io import StringIO
+from typing import ClassVar
+
+import requests
+from core import taskmanager
+from core.schemas import task
+from core.schemas.entity import AttackPattern, Campaign, IntrusionSet, Tool
+
+
+def _get_build_id():
+    response = requests.get("https://threats.wiz.io")
+    if response.status_code != 200:
+        return None
+    p = re.compile('"buildId":"(.*?(?="))"')
+    m = p.search(response.text)
+    if m:
+        return m.group(1)
+    else:
+        None
+
+def _get_properties(build_id, property):
+    base_uri = f"https://threats.wiz.io/_next/data/{build_id}/all-{property}"
+    main_endpoint = f"{base_uri}.json?page=all-{property}"
+    response = requests.get(main_endpoint)
+    properties = dict()
+    if response.status_code != 200:
+        return properties
+    data = response.json()
+    for record in data.get('pageProps').get('records').get('block'):
```

Comment:
Updated all functions outside the feed task to directly access keys. Since these are handling the feed parsing if there's a problem, we will catch the exception that will be part of logs / UI.